js/filter_type.js

```diff
@@ -18,16 +21,19 @@ $(document).ready(function(){
                 var dist_options = [];
 
                 if (sortOptions.includes(xAxis) && sortOptions.includes(yAxis) && (xAxis !== yAxis)){
+                    var submitButtonTxt = "<div id='sort-form'></div></br><button type='button' id='submit-scroll' class='btn' onclick='sortPlots()'>";
+                    submitButtonTxt += "&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspSUBMIT &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</button>";
+                    $("#submit-button").html(submitButtonTxt);
                     sortOpt.splice(sortOpt.indexOf(xAxis), 1);
                     sortOpt.splice(sortOpt.indexOf(yAxis), 1);
                     var filtersStr = "";
                     var lookup = {};
                     var result = [];
 
                     for (var l = 0; l < sortOpt.length; l++){
-                        filtersStr += "<label>Filter" + (l + 1) + "&nbsp;</label>";
-                        filtersStr += "<select id='filter" + (l + 1) + "'>";
-                        filtersStr += "<option disabled selected value> - " + sortOpt[l] + " - </option>";
+                        filtersStr += "<label class='col-md-4 control-label'>Filter" + (l + 1) + "&nbsp;</label>";
+                        filtersStr += "<select id='filter" + (l + 1) + "' class='form-control selectpicker'>";
+                        filtersStr += "<option selected value='justNull'> - " + sortOpt[l] + " - </option>";
```

Comment:
Why is this value `justNull`, it not immediately obvious to me.  Should it be actually just `null`, or blank, i.e.: `""`?

Reply:
Done